assets/prototype/domain/shared/predicates/priceTypes/selectionPredicates/index.test.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * Internal dependencies
+ */
+import { PriceBasetype } from '../../../../../domain/eventEditor/data/types';
+import {
+	isBasePrice,
+	isNotBasePrice,
+	isDiscount,
+	isNotDiscount,
+	isPercent,
+	isNotPercent,
+	isTax,
+	isNotTax,
+	isFlatFeeSurcharge,
+	getDefaultPriceModifierType,
+} from './index';
+import { nodes as priceTypes } from '../../../../../domain/eventEditor/data/queries/priceTypes/test/data';
+
+describe('isBasePrice & isNotBasePrice', () => {
+	it('should return true if priceType is base price', () => {
+		priceTypes.forEach((priceType) => {
+			if (priceType.isBasePrice === true) {
+				expect(isBasePrice(priceType)).toBe(true);
+			}
```

Comment:
should maybe add an else clause to confirm that the function is not sending false positives

ie:

```js
	if (priceType.isBasePrice === true) {
		expect(isBasePrice(priceType)).toBe(true);
	} else {
		expect(isBasePrice(priceType)).toBe(false);
	}
```

or you could just compare directly against `priceType.isBasePrice`:


```js
	expect(isBasePrice(priceType)).toBe(priceType.isBasePrice);
```

maybe do the same for all of the tests using this pattern

Reply:
I like this suggestion:
`expect(isBasePrice(priceType)).toBe(priceType.isBasePrice);`
it looks a bit cleaner
thx :bowing_man:

Reply:
Updated: https://github.com/eventespresso/event-espresso-core/pull/2196/commits/07c28fb1b222c22f2271c489d1ed88be17491490